### PR TITLE
LPS-144149 Fix jaxb impl provision for java 11

### PR DESCRIPTION
**Pull request by rafaprax**

### CODE REVIEW

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:help

**Comment by rafaprax** [comment]
ci:test:jdk11

**Comment by rafaprax** [comment]
ci:test:startup

**Comment by rafaprax** [comment]
ci:test:jdk11

File: modules/apps/portal-remote/portal-remote-soap-extender-impl/build.gradle [hunk 1]
```diff
@@ -1,6 +1,7 @@
 dependencies {
 	compileOnly group: "com.liferay", name: "biz.aQute.bnd.annotation", version: "4.2.0.LIFERAY-PATCHED-2"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
+	compileOnly group: "com.sun.xml.bind", name: "jaxb-osgi", version: "2.3.4"
```
Comment (rafaprax) [inline modules/apps/portal-remote/portal-remote-soap-extender-impl/build.gradle:4]: #todo
It is not needed. Remove this after tests finish

**Comment by catarineaguiar** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
@rafaprax no regressions found

**Pull request closed without merging**